MiniAn/minian_definitions.py

```diff
@@ -119,8 +119,7 @@ class Preview:
         MAX_PROJ_DATASET_NAME   = "MaxProjDatasetName"
 
     class CrossReg:
-            """
-            Cross Registration keys
-            """
-
-            _ = ""
\ No newline at end of file
+        FNAME_CROSSREG          = "fname_crossReg"
+        H5PATH_IMAGES_CROSSREG  = "h5path_crossRegImages"
+        H5PATH_ROI_CROSSREG     = "h5path_crossRegRoi"
+        CROSS_REGISTRATION      = "CrossRegister"
```

Comment:
linked to [this comment](https://github.com/doriclenses/pyDoric/pull/26/files#r1362293146) I think the text should all be camel Case